web/src/main/java/no/nav/modiapersonoversikt/service/skatteetaten/innkreving/InnkrevingskravService.kt

```diff
@@ -135,7 +52,7 @@ private fun Kravlinje.toDomain(): KravPostering =
         opprettetDato = LocalDateTime.now(),
     )
 
-private fun Innkrevingskrav.toDomain(): Krav =
+private fun Innkrevingskrav.toDomain(arenaInfotrygdApi: ArenaInfotrygdApi): Krav =
```

Comment:
Mangler mapping fra ident/orgnr til org/personinfo?

Kanskje gjøre dette til en metode på servicen også?

Reply:
Hvorfor tas APIet inn her? Virker ikke som det brukes. Om tanken er at den vil tilgjengeligjøre informasjon vi trenger for å mappe fra Innkrevingskrav-typen til Krav-typen i fremtiden, så ville jeg heller sendt inn den faktiske dataen vi trenger, ikke APIet.

Reply:
Jeg syns også det hadde vært nice om konverteringen skjedde i API-laget, så service-laget kun må forholde seg til domeneklassene